hack/dind

```diff
@@ -25,6 +25,12 @@ if ! mountpoint -q /tmp; then
 	mount -t tmpfs none /tmp
 fi
 
+# Switch to use iptables instead of nftables (to match the host machine)
+update-alternatives --set iptables  /usr/sbin/iptables-legacy  || true
+update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy || true
+update-alternatives --set arptables /usr/sbin/arptables-legacy || true
```

Comment:
```
00:00:31.648 update-alternatives: using /usr/sbin/iptables-legacy to provide /usr/sbin/iptables (iptables) in manual mode
00:00:31.657 update-alternatives: using /usr/sbin/ip6tables-legacy to provide /usr/sbin/ip6tables (ip6tables) in manual mode
00:00:31.661 update-alternatives: error: alternative /usr/sbin/arptables-legacy for arptables not registered; not setting
00:00:31.662 update-alternatives: error: alternative /usr/sbin/ebtables-legacy for ebtables not registered; not setting
```

AFAIK, `ebtables` and `arptables` are separate utilities, they are not part of iptables (or nftables) and there are no "legacy" versions of them.

Reply:
In the pre-`nft` world, they were separate, but `nft` manages all of them in a unified way, so these other `legacy` versions only exist if the respective packages are installed:

- https://packages.debian.org/buster/amd64/arptables/filelist
- https://packages.debian.org/buster/amd64/ebtables/filelist